h/groups/test/views_test.py

```diff
@@ -155,6 +155,20 @@ def test_create_with_non_ascii_name():
     views.create(_mock_request(params={"name": u"☆ ßüper Gröup ☆"}))
 
 
+@create_fixtures
+def test_create_broadcasts_join_event(Group):
+    group = mock.Mock(hashid=mock.sentinel.hashid)
+    Group.return_value = group
+    request = _mock_request()
+
+    views.create(request)
+
+    request.get_queue_writer().publish.assert_called_once_with('user', {
+        'type': 'group-joined',
```

Comment:
For the sake of simplicity (and consistency with route names, method names, etc.), perhaps this event type should be an imperative rather than a past-participle: `group-join`, `group-leave`.